Horrific hack to make the FFmpeg media plugin link to the static OpenSSL libs.
--- a/qtmultimedia/src/plugins/multimedia/ffmpeg/CMakeLists.txt	2026-01-23 23:53:56.153499238 -0500
+++ b/qtmultimedia/src/plugins/multimedia/ffmpeg/CMakeLists.txt	2026-01-23 23:56:00.993573697 -0500
@@ -4,6 +4,10 @@
 qt_find_package(EGL)
 qt_find_package(VAAPI COMPONENTS VA DRM PROVIDED_TARGETS VAAPI::VAAPI MODULE_NAME multimedia QMAKE_LIB vaapi)

+if (NOT WIN32)
+    link_directories(BEFORE $ENV{OPENSSL_DIR}/lib)
+endif()
+
 qt_internal_add_module(FFmpegMediaPluginImplPrivate
     STATIC
     INTERNAL_MODULE
